oci/pack.go

```diff
@@ -17,7 +17,6 @@ import (
 
 	"github.com/google/go-containerregistry/pkg/crane"
 	"github.com/google/go-containerregistry/pkg/name"
-	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
This commit (a1967b153196a4e4966fd0440d32a77b7d2c6e06) seems to only remove this package instead of "[Adding] method to create packages from index specifications".

Reply:
![image](https://github.com/unikraft/kraftkit/assets/46280801/8ba83aa2-3f90-4b07-bd7b-14a3f893fbc7)

something really strange happened with that line in commits, can't explain but I'll pass through the history again

I lost 20 minutes on that single line trying to figure it out :D

Reply:
weird it shows this for me too but still showed up in the git commit history on github? 🤷

Reply:
removed it